Makefile

```
@@ -87,6 +87,9 @@ dev.provision.services.%: ## Provision specified services with local mounted dir
 
 dev.provision: | check-memory dev.clone.ssh dev.provision.services stop ## Provision dev environment with all services stopped
 
+dev.provision.programs: ## Provision programs
```

```suggestion
dev.cache-programs: ## Copy programs from Discovery to Memcached for use in LMS.
```
In the context of devstack, "provisioning" has come to mean "do one-time setup that persists between container restarts".